main.go

```diff
@@ -1,148 +1,42 @@
 package main
 
 import (
-	"flag"
 	"net/http"
-	"os"
 	"os/user"
-	"runtime"
 
-	"github.com/fabric8-services/fabric8-common/configuration"
 	"github.com/fabric8-services/fabric8-common/log"
-	"github.com/fabric8-services/fabric8-common/metric"
-	"github.com/fabric8-services/fabric8-common/sentry"
 	"github.com/goadesign/goa"
-	goalogrus "github.com/goadesign/goa/logging/logrus"
 	"github.com/goadesign/goa/middleware"
 	"github.com/goadesign/goa/middleware/gzip"
 	"github.com/golang-starters/golang-rest-http/app"
 	"github.com/golang-starters/golang-rest-http/controller"
-	"github.com/google/gops/agent"
-	"github.com/prometheus/client_golang/prometheus"
-	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
 func main() {
 
-	// --------------------------------------------------------------------
-	// Parse flags
-	// --------------------------------------------------------------------
-	var configFilePath string
-	var printConfig bool
-	flag.StringVar(&configFilePath, "config", "", "Path to the config file to read")
-	flag.BoolVar(&printConfig, "printConfig", false, "Prints the config (including merged environment variables) and exits")
-	flag.Parse()
-
-	// Override default -config switch with environment variable only if -config switch was
-	// not explicitly given via the command line.
-	configSwitchIsSet := false
-	flag.Visit(func(f *flag.Flag) {
-		if f.Name == "config" {
-			configSwitchIsSet = true
-		}
-	})
-	if !configSwitchIsSet {
-		if envConfigPath, ok := os.LookupEnv("F8_CONFIG_FILE_PATH"); ok {
-			configFilePath = envConfigPath
-		}
-	}
-
-	config, err := configuration.New(configFilePath)
-	if err != nil {
-		log.Panic(nil, map[string]interface{}{
-			"config_file_path": configFilePath,
-			"err":              err,
-		}, "failed to setup the configuration")
-	}
-
-	if printConfig {
-		os.Exit(0)
-	}
-
-	// Initialized developer mode flag and log level for the logger
-	log.InitializeLogger(config.IsLogJSON(), config.GetLogLevel())
-
-	// Initialize sentry client
-	haltSentry, err := sentry.InitializeSentryClient(
-		nil, // will use the `os.Getenv("Sentry_DSN")` instead
-		sentry.WithRelease(app.Commit),
-		sentry.WithEnvironment(config.GetEnvironment()),
-	)
-	if err != nil {
-		log.Panic(nil, map[string]interface{}{
-			"err": err,
-		}, "failed to setup the sentry client")
-	}
-	defer haltSentry()
-
 	printUserInfo()
 
 	// Create service
 	service := goa.New("golang-foo")
 
 	// Mount middleware
 	service.Use(middleware.RequestID())
-	// Use our own log request to inject identity id and modify other properties
 	service.Use(gzip.Middleware(9))
```

Comment:
Do we really need this for an example that shall just show how to start?

Reply:
We don't really need it. I thought it would be okay to keep it. Shall we remove it?